tests/pdk/finfet_pdk/test_placement_grid.py

```diff
@@ -1,16 +1,24 @@
-import os
 import json
+import shutil
+import pytest
 import textwrap
 from .utils import get_test_id, build_example, run_example
 from . import circuits
+from align.pdk.finfet import MOSGenerator
 
 
-def test_place_on_grid():
-    name = f'ckt_{get_test_id()}'
-    netlist = circuits.tia(name)
-    constraints = []
-    example = build_example(name, netlist, constraints)
-    run_example(example, cleanup=True, n=8)
+@pytest.fixture(autouse=True)
```

Comment:
`autouse` I guess means apply this fixture to every thing in the file. It might be easier to understand what is going on if you explicitly add this fixture to each test (I think that just means make it a parameter.) If not, then this is okay. I was just looking through this file to see how this is working and I couldn't figure it out.

Reply:
That is correct. This allow me to use this fixture and also use `pytest.mark.parametrize`. Is it possible to combine the two?

Reply:
By the way, all tests in `test_placement_grid.py` are supposed to test PlaceOnGrid.

Reply:
I think you can just add the fixture and the parametrize.

Reply:
Yes, 

```
@pytest.fixture()
def place_on_grid(monkeypatch):
    monkeypatch.setenv('blah', 'blah')

@pytest.mark.parametrize('vt', ['N', 'P'])
def test_foo(place_on_grid, vt):
    ...
```
should work just fine. (But this isn't important. Just another way to do it.)